app/bundles/EmailBundle/Tests/EventListener/MatchFilterForLeadTraitTest.php

```diff
@@ -207,6 +207,35 @@ public function dataForInNotInOperatorFilter(): iterable
             false,
         ];
     }
+
+    public function testMatchFilterForLeadTraitEmptyCompany(): void
+    {
+        error_reporting(E_ALL);
```

Comment:
Is this necessary? It will affect other tests as well.

Reply:
This is not necessary, it was removed in the last commit.